homeassistant/components/light/lutron.py

```diff
@@ -0,0 +1,94 @@
+"""Support for Lutron lights."""
+import logging
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, Light
+from homeassistant.components.lutron import (
+    LutronDevice, LUTRON_DEVICES, LUTRON_GROUPS, LUTRON_CONTROLLER)
+from homeassistant.util.async import (
+    run_coroutine_threadsafe)
+
+DEPENDENCIES = ['lutron']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup Lutron lights."""
+    area_devs = {}
+    devs = []
+    for (area_name, device) in hass.data[LUTRON_DEVICES]['light']:
+        dev = LutronLight(hass, area_name, device,
+                          hass.data[LUTRON_CONTROLLER])
+        area_devs.setdefault(area_name, []).append(dev)
+        devs.append(dev)
+    add_devices(devs)
```

Comment:
Ok, I sorted it out. Turns out I don't know how to read. Sigh. I have to manually create entity_ids since I'm being called before they are assigned (just like self.hass). Changed to manually set entity ids and now can use add_devices(..., True), etc. PTAL.